### ISSUE TITLE
Build instructions don't work for me

### ISSUE THREAD

**Issue by fransflippo** [swisnl/jQuery-contextMenu#598]
Running on Mac OS X Sierra. I installed a fresh "yarn" using Homebrew - yarn 1.3.2.

Running `yarn run webpack` as per the `README.md` results in the following:
    yarn run v1.3.2
    error Command "webpack" not found.
    info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by bbrala** [comment]
Hi, are you trying to build the 3.x beta? The normaal release uses gulp.

**Comment by bbrala** [comment]
If you see, could you try `yarn run webpack-build`, does that work?

**Comment by fransflippo** [comment]
Yes, the 3.x beta. `yarn run webpack-build` does work (well, I get a `/bin/sh: webpack: command not found, but that would just be a matter of installing `webpack`, I suppose). Thanks!

**Comment by bbrala** [comment]
hmm weird. When using yarn to run webpack it should add the modules you have available in the project. Which means webpage since that is in the dev dependencies. Perhaps I can check a mac of a colleague sometime soon to see if things work normally or not.

**Comment by bbrala** [comment]
Tried it on linux and Windows, both seem fine. Still haven't had a free mac to do so some testing.

**Comment by Skillvendor** [comment]
Same issue. With yarn 1.7.0 I get the command not found error
